src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -7,20 +7,21 @@ import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
 
 class Conf(args: Seq[String]) extends ScallopConf(args) {
   printedName = "easy-stage-dataset"
-  val indent_____ = printedName.replaceAll(".", " ")
   version(s"$printedName v${Version()}")
+  private val padding = Array.fill(printedName.length)(' ').mkString("")
   banner(s"""
            |Stage a dataset in EASY-BagIt format for ingest into an EASY Fedora Commons 3.x Repository.
            |
            |Usage:
            |
            | $printedName -t <submission-timestamp> -u <urn> -d <doi> [ -o ] \\
-           | $indent_____    <EASY-bag> <staged-digital-object-set>
+           | ____________ -b <EASY-bag> -s <staged-digital-object-set>
            |
            |Options:
-           |""".stripMargin)
+           |""".stripMargin.replaceAll("_+",padding))
```

Comment:
@jo-pol 
If you don't like my first solution, how about just naming the "padding" value ``_________``; then the `replaceAll` is no longer necessary and it still looks almost the same.